go/grpcwebproxy/backend.go

```diff
@@ -103,5 +119,19 @@ func buildBackendTlsOrFail() *tls.Config {
 			}
 		}
 	}
+	if *flagBackendTlsClientCertRequired {
+		if *flagBackendTlsClientCert == "" {
```

Comment:
Change this check to just check the other two are set.

Reply:
Added a bit of logic to make sure they are both set and removed the flag